docker-compose.yaml

```diff
@@ -1,6 +1,6 @@
 version: '3'
 services:
-  vitalam-node:
+  dscp-node:
     image: ghcr.io/digicatapult/vitalam-node:v2.11.0
```

Comment:
need to wait until `vitalam-node` is renamed and built until we change or test workflow fails

Reply:
same applies to `helm-lint` and `helm-test` in workflows which point to the `node` e.g. `ct lint --validate-maintainers false --charts helm/${{ needs.repo_ids.outputs.repo_name }}/ --chart-repos vitalam-node=https://digicatapult.github.io/vitalam-node/`